platform/vs/docker-sonic-vs/Dockerfile.j2

```diff
@@ -157,6 +157,8 @@ COPY ["port_breakout_config_db.json", "/etc/sonic/"]
 COPY ["database_config.json", "/etc/default/sonic-db/"]
 COPY ["hostname.j2", "/usr/share/sonic/templates/"]
 COPY ["default_chassis_cfg.json", "/etc/default/sonic-db/"]
+COPY ["asic_table.json", "/etc/sonic/"]
```

Comment:
What about the peripheral_table for vs?

Reply:
If there is no gear box installed we don’t need that

Reply:
In the docker-sonic-vs/buffermgrd.sh, 'peripheral_table.json' is one of the mandatory args for starting buffermgr in dynamic mode

Reply:
Will fix it.